### DISCUSSION TITLE
How to fix the map?

### DISCUSSION THREAD

**Discussion by msbarry** [OvertureMaps/data#42] [Discussion]
Browsing around the [map I made from this first overture release](https://msbarry.github.io/planetiler-overture-demo/), I can see some data issues. Since OpenStreetMap is the editable data source that feeds into Overture, I assume the process to fix the map would be to add the corrected feature in OpenStreetMap, but is there a way to way to reliably link an element you add to OpenStreetMap to an Overture element? Maybe something like include the overture ID in a tag in OSM? Or even better a way to pre-populate a new feature in OSM from the Overture feature you want to correct?

**Comment by louwers** [comment]
I am also wondering about this. There are POIs that have long closed or moved that Overture maps is "0.99 confident" about. They are correct on OSM.

**Reply by sethfitz** [reply]
FYI, Overture POIs are not derived from OSM.

**Reply by louwers** [reply]
Ah I thought OSM was also used as a data source. Thanks for the info.

**Reply by Cj-Malone** [reply]
Maybe OSM should be included in the mix. There is already some kind of system to join MS and FB POIs. Adding OSM would surely improve the combined dataset

**Reply by msbarry** [reply]
I filed the question under "places" but really it applies to all of the layers. Hopefully the plan is to eventually join osm poi's into the place layer as well.

**Reply by Cj-Malone** [reply]
- Roads/"Transportation" looks like it's mainly OSM. Presumably fixing it in OSM will fix it in future Overture releases.
- Buildings includes OSM buildings, [Microsoft ML](https://github.com/microsoft/GlobalMLBuildingFootprints) and Esri. Presumably it's like Daylight, where OSM buildings take priority. Fix it in OSM.
- POIs/"Places" looks to be made of Facebook and MS POIs. Doubt either of those accept fixes.
- Administrative Boundaries, I don't know the source of this layer, but I doubt it's fixable by us. Presumably Overture would make fixes, as long as they fix in the US world view.

Looking at the sources going into Overture, it looks like OSM is the only read/write source.

**Comment by cbeddow** [comment]
My ideal personal view would be that the POIs go into Rapid as a layer, and if you reject the POI it pings an API that records the ID and a rejection reason like "doesn't exist" or "duplicate" or "wrong name" etc. And if you add it/merge it with existing, then it merges the Overture ID and using OSMCha you could see the difference if it gets moved or edited after accepting. 

Not sure if this is the best way to go but hoping to drive discussion on this.

**Reply by Cj-Malone** [reply]
Rapid developers have shown interest in giving the rejected data back to suppliers.

**Reply by msbarry** [reply]
It looks like at least the first part of your proposal is supported now. Rapid just added support for showing pmtiles layers: https://github.com/facebook/Rapid/pull/1016 so you could add the poi pmtiles listed in that PR as an overlay, or the one I made with all of the layers on https://msbarry.github.io/planetiler-overture-demo/

**Reply by bhousel** [reply]
Yes! I was able to get it to work by pointing to the `boston.pmtiles` that's checked into GitHub:
`https://raw.githubusercontent.com/msbarry/planetiler-overture-demo/main/boston.pmtiles`

<img width="1671" alt="Screenshot 2023-08-11 at 9 46 54 PM" src="https://github.com/OvertureMaps/data/assets/38784/4db1eec4-0974-491c-adbc-7f541439e993">
